### ISSUE TITLE
All imports in public should be relative

### ISSUE THREAD

**Issue by wallw-bits** [jakedeichert/svelvet#79]
if I do `npm run build` for the production build, I expect to be able to take public and run it anywhere: from a deeply-nested folder in a document root (e.g. example.com/path/to/myapp) on a static file server or merely from File > Open. Currently all the imports use `/dist/...` which break those cases. If they were all relative to the file requesting them that would help significantly.

This is less important for `npm run dev` but I believe it would still work fine with relative imports.

**Comment by jakedeichert** [comment]
Yeah, i agree that all imports should probably be relative. Off the top of my head, I can't think of any issues that would cause... but it will require some investigation 👍
